doc/source/whatsnew/v0.24.1.rst

```diff
@@ -50,16 +50,19 @@ The `sort` option for :meth:`Index.intersection` has changed in three ways.
 Fixed Regressions
 ~~~~~~~~~~~~~~~~~
 
-- Bug in :meth:`DataFrame.itertuples` with ``records`` orient raising an ``AttributeError`` when the ``DataFrame`` contained more than 255 columns (:issue:`24939`)
-- Bug in :meth:`DataFrame.itertuples` orient converting integer column names to strings prepended with an underscore (:issue:`24940`)
+- Fixed regression in :meth:`DataFrame.to_dict` with ``records`` orient raising an
```

Comment:
maybe worthwhile to just classify all of these into the appropriate categories

Reply:
If you mean adding "subsections": I personally don't find that needed, since it are only a few items.